USERMANUAL.md

```diff
@@ -604,3 +604,14 @@ CloudDQ supports the follow methods for authenticating to GCP:
 1. OAuth via locally discovered [Application Default Credentials (ADC)](https://cloud.google.com/docs/authentication/best-practices-applications#overview_of_application_default_credentials) if only the arguments `--gcp_project_id`, `--gcp_bq_dataset_id`, and `--gcp_region_id` are provided.
 2. Using an [exported JSON service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) if the argument `--gcp_service_account_key_path` is provided.
 3. [Service Account impersonation](https://cloud.google.com/iam/docs/impersonating-service-accounts) via a source credentials if the argument `--gcp_impersonation_credentials` is provided. The source credentials can be obtained from either `--gcp_service_account_key_path` or from the local ADC credentials.
+
+### Materialization Hours
+1. Configure the bigquery job intermediate tables expiration hours in the provided dataset `--gcp_bq_dataset_id`.
+2. if the CLI argument `--intermediate_table_expiration_hours` is defined, CloudDQ will set the same as expiration hours for intermediate tables. 
+  `--intermediate_table_expiration_hours` is currently an optional argument and has a default value of `24 hours`.
+
+### Number of Concurrent Bigquery Threads
+1. CLI argumnet `--num_threads` specifies the number of concurrent bigquery operations that can be increased to reduce run-time. 
+    We advice setting this to number of cores of your run-environment machines. 
+2. One worker thread per core seemed a good number for how many threads to run at once. This number is chosen much more carefully based on other factors, such as other applications and services running on the same machine.
+3. if the CLI argument `--num_threads` is defined, CloudDQ will limit the number of threads to this value. `--num_threads` is currently an optional argument and has a default value of `8 threads`.
```

Comment:
I think it might be a little more readable if we refactor this section to 2 points

1. Explain what `--num_threads` is/does
2. Explain what the default value is, how to change if intended, and how we suggest user to set the value (e.g. the number of cores of the runtime environment also with consideration of the impact on other workloads.)

Reply:
updated the doc